pvlib/albedo.py

```diff
@@ -0,0 +1,137 @@
+"""
+The ``albedo`` module contains functions for modeling albedo.
+"""
+
+from pvlib.tools import sind
+import numpy as np
+import pandas as pd
+
+
+WATER_COLOR_COEFFS = {
+    'clear_water_no_waves': 0.13,
+    'clear_water_ripples_up_to_2.5cm': 0.16,
+    'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps': 0.23,
+    'clear_water_frequent_whitecaps': 0.3,
+    'green_water_ripples_up_to_2.5cm': 0.22,
+    'muddy_water_no_waves': 0.19
+}
+
+WATER_ROUGHNESS_COEFFS = {
+    'clear_water_no_waves': 0.29,
+    'clear_water_ripples_up_to_2.5cm': 0.7,
+    'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps': 1.25,
+    'clear_water_frequent_whitecaps': 2,
+    'green_water_ripples_up_to_2.5cm': 0.7,
+    'muddy_water_no_waves': 0.29
+}
+
+
+def inland_water_dvoracek(solar_elevation, surface_condition=None,
+                          color_coeff=None, wave_roughness_coeff=None):
+    r"""
+    Estimation of albedo for inland water bodies.
+
+    The available surface conditions are for inland water bodies, e.g., lakes
+    and ponds. For ocean/open sea, see
+    :const:`pvlib.irradiance.SURFACE_ALBEDOS`.
+
+    Parameters
+    ----------
+    solar_elevation : numeric
+        Sun elevation angle. [degrees]
+
+    surface_condition : string, optional
+        If supplied, overrides ``color_coeff`` and ``wave_roughness_coeff``.
+        ``surface_condition`` can be one of the following:
+
+        * ``'clear_water_no_waves'``
+        * ``'clear_water_ripples_up_to_2.5cm'``
+        * ``'clear_water_ripples_larger_than_2.5cm_occasional_whitecaps'``
+        * ``'clear_water_frequent_whitecaps'``
+        * ``'green_water_ripples_up_to_2.5cm'``
+        * ``'muddy_water_no_waves'``
+
+    color_coeff : float, optional
+        Water color coefficient. [-]
+
+    wave_roughness_coeff : float, optional
+        Water wave roughness coefficient. [-]
+
+    Returns
+    -------
+    albedo : numeric
+        Albedo for inland water bodies. [-]
+
```

Comment:
```suggestion

    Raises
    ------
    ValueError
        If neither of ``surface_condition`` nor the pair ``color_coeff`` and ``wave_roughness_coeff`` are given.
        If ``surface_condition`` and any of ``color_coeff`` or ``wave_roughness_coeff`` are given.
        These parameters are mutually exclusive.

    KeyError
        If ``surface_condition`` is invalid.

```
Feel free to use a more natural language.
Maybe you can note the exclusiveness of the optional parameters elsewhere too.